cockroachdb/hatch.toml

```diff
@@ -10,4 +10,5 @@ matrix.version.env-vars = [
   { key = "COCKROACHDB_VERSION", value = "latest", if = ["latest"] },
 ]
 
-[envs.bench]
+[envs.bench.env-vars]
+COCKROACHDB_VERSION = "latest"
```

Comment:
Chiming in with reproducible build paranoia again :D
Will this introduce unexpected breakages into our daily CI workflow?

Reply:
Yup, you're right. My goal here is to fix the bench env and don't want to pin this in this PR. I'm opening a separate PR to pin this version and add a `latest` env if it works right away, otherwise I'll add a card to our backlog. What do you think?

Reply:
It works! https://github.com/DataDog/integrations-core/pull/13318

Reply:
👍